.schema/latest.json

```diff
@@ -50,6 +50,7 @@
         "git_release_type": null,
         "git_tag_enable": null,
         "git_tag_name": null,
+        "one_pr_per_package": null,
```

Comment:
For the name, what do you think about `pr_per_package`?

Reply:
That does sound better